### ISSUE TITLE
[NTR]: X chromosome 

### ISSUE THREAD

**Issue by mikewhitelab** [Tree-of-Sex/tos-ontology#11]
### In which category is your new term request?

- [ ] new_definition
- [X] existing_definition

### Preferred term label

X chromosome

### Textual definition

The sex chromosome present in both sexes of species in which the male is the heterogametic sex. Two copies of the X chromosome are present in each somatic cell of females and one copy is present in males.
go:0000805

### Suggested parent term

sex chromosome
chromosome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by laiamaringual** [comment]
I agree with the definition "The sex chromosome present in both sexes of species in which the male is the heterogametic sex". I would move the following information "Two copies of the X chromosome are present in each somatic cell of females and one copy is present in males. go:0000805" from the definition to the additional context section.
